README.md

```diff
@@ -66,7 +66,39 @@ The application is built automatically and published on [Cozy Registry](https://
 
 ### Android environment
 
-To be able to start the mobile app on an emulator or a real device, you can follow [this guide](https://gist.github.com/drazik/11dfe2014a6b967821df93b9e10353f4).
+First you need to export your local host IP address:
+
+```bash
+$ export DEV_HOST=[YOUR_LOCAL_IP_ADDRESS]
+```
+
+And add some specifiations in `src/targets/mobile/config.xml` inside `<platform name="android">` after `<config-file parent="/manifest/application" target="app/src/main/AndroidManifest.xml"></config-file>`:
+
```

Comment:
I don't think this is the right way to fix the issue. If you build a release from your computer you'll forget to remove this line.

Also this is needed only for Android 9+ device. 

I think the real answer to this issue is to implement something like that  https://stackoverflow.com/questions/46345603/cordova-config-xml-environment-variables 


(https://github.com/cozy/cozy-drive/issues/2091 link for posterity)

Reply:
> If you build a release from your computer you'll forget to remove this line.

Bien vu !

Reply:
J'ai essayé différentes approches avec des scripts, et dans tous les cas le fichier `config.xml` est modifié et apparaît donc dans les commit. Pour l'instant je ne vois pas trop comment éviter ça.

Pour la partie build une release, Idéalement il faudrait avoir un script qui s'occupe de vérifier ce qu'on fait (build pour la prod ou pour le dev) et de construire le bon `config.xml`. Je pourrai revenir sur ce point plus tard si j'ai du temps, pour l'heure je pense qu'un ajout dans la doc peut suffire, non ?

Reply:
On peut ajouter une icone ⚠️  dans un premier temps puis faire une issue ?

Reply:
Ou ça l'icone ? Il y a déjà dans la doc un `:warning: Be careful to not commit this modification, and to not build a release with it. It's for developing purpose only. `